Add a post about Igalia joining the GC (#107)
openwebdocs/owd-website: pages/content/posts/igalia-gc.md

---
author: "Florian Scholz"
title: "Igalia Joins Open Web Docs Governing Committee"
date: "2024-03-18"
tags: ["announcements"]
ShowToc: false
ShowBreadCrumbs: false
excerpt: "Open Web Docs is excited to welcome Igalia on our Governing Committee joining Google and Microsoft Edge."
---

> Open Web Docs is excited to welcome Igalia on our Governing Committee joining Google and Microsoft Edge. The Governing Committee administers OWD’s financial sponsorships made via Open Collective, and explores strategic partnerships and needs for web documentation projects.

Reviewer: ```suggestion
Open Web Docs is excited to welcome Igalia to our Governing Committee, joining Google and Microsoft Edge. The Governing Committee administers OWD’s financial sponsorships, made via Open Collective, and explores strategic partnerships and needs for web documentation projects.
```

The Governing Committee is composed of organizations sponsoring at the Platinum level, elected representatives from the Gold membership level, individual advisory members, and OWD’s Director.

> Igalia is a cooperative software consultancy focused on open source software known for its contributions and commitments to both open-source and open standards. Igalia is a key contributor to each of the Web engines and often leads the development of many feature implementations that Open Web Docs supports with documentation.

Reviewer: ```suggestion
[Igalia](https://www.igalia.com/about/) is a cooperative software consultancy focused on open-source software. Igalia is known for its contributions and commitments to both open-source and open standards. They are a key contributor to every browser engine and lead the development of many Web feature implementations that Open Web Docs supports with documentation.
```

> “Igalia has been a staunch supporter of Open Web Docs since its inception, and we‘re honored to become part of the Governing Committee. On a more personal level, everything about OWD’s mission resonates with me, so I’m excited by this opportunity to participate more actively in the work.”
– Eric Meyer, Developer Advocate at Igalia

> “Igalia has been a long-time partner to OWD and a big supporter of open web documentation. Igalia has been a key member in our Steering Committee since OWD’s inception in 2021. I’m thrilled to have Igalia on our Governing Committee going forward.”
– Florian Scholz, Director of Open Web Docs

> With a $20k yearly donation, Igalia is an OWD Gold Sponsor and helps us to get closer to our yearly goal to fully fund four full-time Tech Writers to work on Open Web Documentation.

Reviewer: ```suggestion
With a $20k yearly donation, Igalia is an OWD Gold Sponsor, helping us reach our yearly goal of fully funding four full-time technical writers to work on open web documentation.
```
